controllers/util/storageconsumer.go

```diff
@@ -13,7 +13,9 @@ import (
 
 const (
 	// Reserved RadosNamespaceName for internal use and their representation at different layes
-	ImplicitRbdRadosNamespaceName = "<implicit>"
+	ImplicitRbdRadosNamespaceName                = "<implicit>"
+	StorageConsumer418ProcessedAnnotationKey     = "ocs.openshift.io/storageconsumer-4-18-processed"
+	StorageConsumer418ProcessedAnnotationPending = "pending"
```

Comment:
Please remove any annotation logic from this PR. We will introduce similar logic based on feedback coming from the upgrade controller implementation